### PR TITLE
Feat fullduplex meas

### PR DESCRIPTION
**Pull request by obejarano**
Added option to collect measurements to/from all boards in the base station.
Changed reciprocal_calib naming to internal_measurements

### CODE REVIEW

File: CC/Sounder/config.cc [hunk 1]
```diff
@@ -217,7 +228,11 @@ Config::Config(const std::string& jsonfile, const std::string& directory,
 #endif
         }
         slot_per_frame_ = calib_frames_.at(0).size();
-        pilot_slot_per_frame_ = 2; // up and down reciprocity pilots
+        if (ref_node_enable_ == true) {
+            pilot_slot_per_frame_ = 2; // Two pilots (up/down)
+        } else {
+            pilot_slot_per_frame_ = 2 * n_bs_sdrs_[0]; // Two pilots per board (up/down)
```
Comment (rdoosty) [inline CC/Sounder/config.cc:234]: I think this should be `num_channels * (n_bs_sdrs_[0] - 1)`

File: CC/Sounder/config.cc [hunk 2]
```diff
@@ -741,6 +756,16 @@ bool Config::isData(int frame_id, int slot_id)
     }
 }
 
+bool Config::isRx(int frame_id, int slot_id)
+{
+    // Generic RX (assuming cell == 0)
+    try {
+        return calib_frames_[0][frame_id % calib_frames_[0].size()].at(slot_id) == 'R' || calib_frames_[0][frame_id % calib_frames_[0].size()].at(slot_id) == 'P';
```
Comment (rdoosty) [inline CC/Sounder/config.cc:763]: Run code styling here.

File: CC/Sounder/config.cc (hunk 1 above)
Comment (obejarano) [reply CC/Sounder/config.cc]: I decided to leave it as num_channels * n_bs_sdrs_[0]
we end up with an MxM matrix but I think it's easier to map boards to the matrix during analysis

**Review by rdoosty — APPROVED** [review]

File: CC/Sounder/config.cc [hunk 3]
```diff
@@ -741,6 +758,19 @@ bool Config::isData(int frame_id, int slot_id)
     }
 }
 
+bool Config::isRx(int frame_id, int slot_id)
+{
```
[the remark this answers is not in the document]
Comment (obejarano) [reply CC/Sounder/config.cc]: not anymore, I just removed it

**Pull request merged**